wallet/src/util/well-known-petnames.ts

```diff
@@ -0,0 +1,3 @@
+export const wellKnownPetnames = {
+  IbcATOM: 'ATOM',
```

Comment:
I expect the vbank name in production to be ATOM.

We can/should change that in our main configs.

Reply:
Just changed everything to reference 'ATOM' and do the 'IbcATOM' -> 'ATOM' conversion in the beginning when the purse is loaded.